t/t-standalone-file.sh

```diff
@@ -109,17 +109,17 @@ do_local_path_test () {
   cd "$TRASHDIR"
 
   # Check a clone using an absolute Unix-style path.
-  GIT_TRACE=1 GIT_TRANSFER_TRACE=1 git clone "$testdir" "$reponame-repo1"  2>&1 | tee clonecustom.log
+  GIT_TRACE=1 GIT_TRANSFER_TRACE=1 git clone "$testdir$suffix" "$reponame-repo1"  2>&1 | tee clonecustom.log
```

Comment:
Again, my suggestion here would be to pass `$suffix` as a second argument to the function, rather than rely on a global (although of course that does work).